app/Services/LLMService.php

```
@@ -0,0 +1,558 @@
+        // We're only using OpenRouter models, so always use OpenRouter
+        return $this->generateTextWithOpenRouter($prompt, $options);
+    }
+
+    /**
+     * Generate text using deep research models (o3/o4) via v1/responses endpoint
+     */
+    protected function generateWithDeepResearch(string $prompt, array $options = []): string
+    {
+        $model = $options['model'] ?? $this->config['model'] ?? config('ai-models.purposes.fallback');
+        $maxTokens = $options['max_tokens'] ?? $this->config['max_tokens'] ?? 100000;
+        $temperature = $options['temperature'] ?? $this->config['temperature'] ?? 0.7;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$temperature'. (Unused Code Rules)

(UnusedLocalVariable)